Makefile

```diff
@@ -6,4 +6,4 @@ install:
 
 .PHONY: test
 test:
-	godog
```

Comment:
I assume we just want to run `go test` instead of `godog` because we want to just test the framework, don't we? Eventually, we should also add the `godog` in a separate, later stage